lib/core/client.js

```diff
@@ -1 +1 @@
-module.exports = require('./dist/cjs/index');
+module.exports = require('./dist/esm/index');
```

Comment:
It's only for backwards compatibility since `@storybook/core` was refactored into `core-client` and `core-server` in a minor release (6.2) and we didn't know what might point at the `@storybook/core/client` and `@storybook/core/server` entry points